frontend/ui/data/css_tweaks.lua

```diff
@@ -879,26 +872,12 @@ body[name="comments"] > title {
                 ]],
                 separator = true,
             },
-            {
-                id = "fb2_footnotes_regular_font_size",
-                title = _("Keep regular font size"),
-                description = _([[
-FB2 footnotes and endnotes get a smaller font size when displayed in-page. This allows them to be shown with the normal font size.]]),
-                css = [[
-body[name="notes"] > section,
-body[name="comments"] > section
-{
-    -cr-only-if: fb2-document;
-        font-size: 1rem !important;
-}
-                ]],
-            },
             separator = true,
         },
         {
             id = "footnote-inpage_epub",
             conflicts_with = function(id) return util.stringStartsWith(id, "footnote-inpage_") end,
-            global_conflicts_with = function(id) return util.stringStartsWith(id, "footnote-inpage_epub") end,
+            global_conflicts_with = false,
```

Comment:
by the way, is it intentional that this lets you set multiple of these footnote tweaks as global defaults, even if they conflict? You can do that on master and I've preserved the behavior, but I always found it strange. As soon as you then adjust them in book-specific settings you can only keep one, so it's very inconsistent.

Reply:
Indeed, the question is what you mean by conflict exactly. Setting all the footnote classes as default is very reasonable for example.